applications/rag/README.md

```diff
@@ -182,7 +185,7 @@ This step generates the vector embeddings for your input dataset. Currently, the
 2. From [Google Cloud Platform IAP](https://pantheon.corp.google.com/security/iap), check if the allowlisted user has role `IAP-secured Web App User`. This role is necessary to access the application through IAP.
```

Comment:
Line above -- we are expecting an error for the frontend, right? Let's say "you will receive an error and you will have to manually enable IAP". We should also explain why this is manual and why there are IPs being exposed.

Reply:
This one will be fixed